### PR TITLE
diskio.cc: Allow 'partuuid' label usage

### PR DESCRIPTION
**Pull request by lasers**
Addresses #342

### CODE REVIEW

File: src/diskio.cc [hunk 1]
```diff
@@ -92,12 +98,19 @@ struct diskio_stat *prepare_diskio_stat(const char *s) {
...
-  if ((stat(&(stat_name[0]), &sb) != 0) || !S_ISBLK(sb.st_mode)) {
-    NORM_ERR("diskio device '%s' does not exist", &device_s[0]);
+  if (strncmp(s, "label:", 6) == 0) {
+    snprintf(&(stat_name[0]), text_buffer_size.get(*state), "/dev/%s",
+           &(device_name[0]));
+    if ((stat(&(stat_name[0]), &sb) != 0) || !S_ISBLK(sb.st_mode)) {
+      NORM_ERR("diskio device '%s' does not exist", &device_s[0]);
+    }
+  } else if (0 == (strncmp(s, "partuuid:", 9))) {
+    if ((stat(rpbuf2, &sb) != 0) || !S_ISBLK(sb.st_mode)) {
```
[the remark this answers is not in the document]
Comment (lasers) [reply src/diskio.cc]: @su8

[the remark this answers is not in the document]
Comment (su8) [reply src/diskio.cc]: @lasers Been little busy with https://github.com/su8/shufi, sorry.

**Comment by su8** [comment]
Shall I review this to get merged ?

**Comment by lasers** [comment]
@su8 Yes. 
2018-08-10-094533_715x298_scrot

I have to go afk for an hour or two now. Hurry up. ;-)

**Pull request merged**